crucible-syntax/src/Lang/Crucible/Syntax/Concrete.hs

```diff
@@ -112,20 +120,20 @@ printExpr = toText (PrintRules rules)
         rules (Kw Registers) = Just (Special 0)
         rules _ = Nothing
 
-data E s t where
-  EAtom  :: !(Atom s t) -> E s t
-  EReg   :: !Position -> !(Reg s t) -> E s t
-  EGlob  :: !Position -> !(GlobalVar t) -> E s t
-  EDeref :: !Position -> !(E s (ReferenceType t)) -> E s t
-  EApp   :: !(App () (E s) t) -> E s t
+data E s ext t where
```

Comment:
Why does `ext` appear after `s` here? That seems to go against the `ext s` convention used in other data types (`SomeExpr`, `SomeBVExpr`, `Stmt`, `Block`, `CFG`, etc.).

Reply:
Good point.  I fixed this for `E` and `Rand`.